packages/api/cms-api/src/blocks/PixelImageBlock.ts

```diff
@@ -43,6 +43,8 @@ class PixelImageBlockData extends BlockData {
         // eslint-disable-next-line @typescript-eslint/no-unused-vars
         const { createdAt, updatedAt, folder, license, ...data } = file;
 
+        const fileUrl = includeInvisibleContent ? await filesService.createFileUrl(file, previewDamUrls) : undefined;
```

Comment:
This is also used for the site preview... Could this be a problem?

Reply:
not really, only authed users can view the site preview.

alternative: lazy load the url. But I would like to keep it and use it for the new clipbaord dam file copy feature